### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
How it looks:
Screen Shot 2022-04-04 at 10 19 26

**Comment by alexander-nitsche** [comment]
Added new commit regarding additional sources (TER, repository and docs.typo3.org) in composer.json.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Contribution/Index.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Authors
+=======
+
+Initially developed by Kasper Skaarhoj, it was polished by over 100 contributors
+in this area while being part of the TYPO3 Core, and then extracted into its own
+extension by Alexander Stehlik and Benni Mack.
+
+This package is now maintained by a loose group of TYPO3 enthusiasts inside
+the TYPO3 Community. Feel free to contact `Benni Mack`_ for any questions
```
Comment (ohader) [inline Documentation/Contribution/Index.rst:25]: Please avoid pointing to Benni here, refer to the issue tracker instead...

Comment (alexander-nitsche) [reply Documentation/Contribution/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Pull request merged**